platform/openide.util/src/org/openide/util/ChangeSupport.java

```diff
@@ -119,6 +108,7 @@ private void fireChange(ChangeEvent event) {
      *         false otherwise.
      */
     public boolean hasListeners() {
-        return !listeners.isEmpty();
+        
+        return !this.listeners.isEmpty();
```

Comment:
`this` is unnecessary here, right? Why add it then?

Reply:
ok